firefly_client/firefly_client.py

```diff
@@ -182,11 +188,12 @@ def make_lab_client(start_browser_tab=False, html_file=_my_html_file, start_tab=
                 print('     Safari: shows an animation to follow on left side bar')
         return FireflyClient(url=_my_url, html_file=html_file,
                              use_lab_env=True, start_tab=start_tab,
-                             start_browser_tab=start_browser_tab)
+                             start_browser_tab=start_browser_tab,
+                             token=token)
```

Comment:
_from standup_: when you get a token and the protocol is http throw and exception.

Reply:
Added to the `__init__` method.